cli/core/pkg/command/context.go

```diff
@@ -240,11 +240,11 @@ func createContextWithKubeconfig() (context *configapi.Context, err error) {
 	}
 
 	context = &configapi.Context{
-		Name:   ctxName,
+		Name:   strings.Trim(ctxName," "),
```

Comment:
I think if we do a trim, it should be closer to where the variable was set.  For example `ctxName` is used above to check if the context already exists and it will not work properly if there are spaces around the context name.

Also, there is a `strings.TrimSpace()` which may be a little nicer to use.

Reply:
agreed, trimmed all input values after reading.